CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 Mapbox welcomes participation and contributions from everyone.
 # main
 
+## Bug fixes 🐞
```

Comment:
should be included in rc so please rebase on latest main and include in rc section instead of main section